aspnetcore/whats-new/dotnet-AspNetCore.Docs-mod0.md

```diff
@@ -1,84 +1,194 @@
 ---
-title: "ASP.NET Core docs: What's new for June 2023"
-description: "What's new in the ASP.NET Core docs for June 2023."
-ms.custom: June-2023
-ms.date: 07/01/2023
+title: "ASP.NET Core docs: What's new for December 2023"
+description: "What's new in the ASP.NET Core docs for December 2023."
+ms.custom: December-2023
+ms.date: 01/01/2024
 ---
 
-# ASP.NET Core docs: What's new for June 2023
+# ASP.NET Core docs: What's new for December 2023
 
-Welcome to what's new in the ASP.NET Core docs for June 2023. This article lists some of the major changes to docs during this period.
+Welcome to what's new in the ASP.NET Core docs for December 2023. This article lists some of the major changes to docs during this period.
+
+## Miscellaneous
+
+### Updated articles
+
+- <xref:index> - Update introduction-to-aspnet-core.md
 
 ## Blazor
 
 ### Updated articles
 
-- <xref:blazor/tutorials/build-a-blazor-app>  - Blazor ToDo tutorial updates
-- <xref:blazor/tooling>
-  - Tooling article updates
-  - Update Tooling article lists
-  - Drop lists in Tooling article
-- <xref:blazor/project-structure> - Project structure article updates
-- <xref:blazor/host-and-deploy/webassembly> - .NET assembly packaging (Webcil)
+- <xref:blazor/fundamentals/signalr>
+  - Highlight placeholder line
+  - Snippet cross-links 8.0 (2)
+  - Terminology consistency updates
+- <xref:blazor/forms/validation>
+  - Fix casing of "isValid" in form code examples
+  - Snippet cross-links 8.0 (2)
+- <xref:blazor/test> - Cross-link bUnit Oslo '22
+- <xref:blazor/security/webassembly/graph-api>
+  - Remove duplicate words
+  - Razor directive ordering
+- <xref:blazor/blazor-ef-core>
+  - Update examples
+  - Snippet cross-links 8.0 (3)
+- <xref:blazor/components/event-handling>
+  - Update examples
+  - Snippet cross-links 8.0 (2)
+  - Snippet cross-links 8.0
+- <xref:blazor/components/index>
+  - Update examples
+  - Snippet cross-links 8.0 (3)
```

Comment:
I'd remove most of these

Reply:
> I'd remove most of these

Since these are Blazor articles I'll leave that to @guardrex